tests/drivers/regulator/fixed/testcase.yaml

```diff
@@ -5,6 +5,8 @@ common:
   harness: ztest
   harness_config:
     fixture: regulator_loopback
+  integration_platforms:
+    - nrf52840dk_nrf52840
```

Comment:
Not thingy52?

Reply:
Could be both i suppose.

Reply:
Not Thingy52. Thingy's do not have arduino gpio headers.

Reply:
Ah yes, sorry i was thinking in terms of regualtors.